src/js/tech/tech.js

```diff
@@ -322,6 +322,7 @@ class Tech extends Component {
     }
 
     let textTracksChanges = Fn.bind(this, function() {
+      let tracks = this.textTracks();
       let updateDisplay = () => this.trigger('texttrackchange');
```

Comment:
Tested, and you are right. The number of handlers grows each time the mode is set to showing, and doesn't shrink when set to modes other than showing. We should be able to simply move it outside of the textTracksChanges function and we can change the bind to that function.

Is it better to handle this in a separate PR, or the same (with another commit)?